src/__tests__/ParserComponent.test.tsx

```diff
@@ -798,7 +873,7 @@ describe('<ParserComponent /> Unit Tests', () => {
     it('timeline is set to true', async () => {
       // Call the parse method with the fake event
       await comp.instance().parse(event);
-      expect(comp.state('showTimeline')).toBeTruthy();
+      expect(comp.state('showTimeline')).toBeFalsy();
```

Comment:
Why'd you change this? The it block doesn't make sense anymore and shouldn't it be Truthy?